svo (install the zip, not me).xml

```diff
@@ -15251,9 +15251,11 @@ svo.valid.remove_unknownany(&quot;clumsiness&quot;)</script>
                         <colorTriggerBgColor>#000000</colorTriggerBgColor>
                         <regexCodeList>
                             <string>You lash out clumsily.</string>
+                            <string>^You lash out at (\w+) with (.+), but miss\.$</string>
```

Comment:
I don't know how much we care about regex performance, in general, but it's probably better in principle to use lazy quantifiers with the more flexible wildcards, like "dot". That is, ".+?" instead of ".+", in this case. Also no need to save the match groups here, but no particular problem with having them there, either.